### ISSUE TITLE
[ERROR][logstash.inputs.logstashinputazureblob] Unknown setting 'azure_blob_file_path_field' for azureblob

### ISSUE THREAD

**Issue by Marusyk** [Azure/azure-diagnostics-tools#199]
> [ERROR][logstash.inputs.logstashinputazureblob] Unknown setting 'azure_blob_file_path_field' for azureblob

logstash-input-azureblob (0.9.13)

I found this commit: https://github.com/Azure/azure-diagnostics-tools/commit/c357e086e2df8532b4ef9e4e35b5b84efd143e4b

@cscheib how it should work?

**Comment by cscheib** [comment]
Are you sure that the release you're running contains this change?  You may have to pull and use what's in `master` here.

It's ruby, so you should be able to figure out where the gem you have now was installed and inspect the contents to see if my work is in there... it's really pretty simplistic.  A boolean to enable the feature, and a string value for the name of the field.

**Comment by Marusyk** [comment]
I don't know ruby! I just want to use this plugin with that setting.

> Are you sure that the release you're running contains this change?
No, I'm not sure, this is the reason why I create the issue. It seems that it doesn't, but I'm using the latest version. So, maybe, someone should release a new version with that setting ?

**Comment by Marusyk** [comment]
any news?

**Comment by cscheib** [comment]
I have no direct affiliation with this project, beyond submitting a pull request to add a feature I used at one time to debug something.  I implemented the feature similarly to how it was implemented elsewhere.

It's possible your release doesn't contain this change?  I don't know.

This setting is kind of "use at your own risk" at this point.

I can't provide support for this, as I don't really have the time or focus right now.  I don't even use this product any more.  My recommendation is to roll up your sleeves and learn some ruby.  It honestly isn't that difficult to learn enough to debug this particular issue.

I'm unsubscribing myself from this issue so as to not get notifications for it any longer.  I wish you good luck!
